src/utils/tokens.ts

```diff
@@ -352,7 +358,7 @@ const tokens = {
     }
 
     if (createAnotherMelt) {
-      const newAddress = await storage.getCurrentAddress();
+      const newAddress = meltAuthorityAddress || await storage.getCurrentAddress();
```

Comment:
Should we validate that `meltAuthorityAddress` belongs to this wallet?

Reply:
I don't think so, this is not like a change address in my opinion, you might want to create an authority output for another wallet. What do you think?

Reply:
I feel that most of the cases it would be a mistake. Maybe add another option such as `--allow-other-wallet-addresses`? What do you think?